src/sentry/static/sentry/app/components/group/tagDistributionMeter.jsx

```diff
@@ -159,15 +164,94 @@ const TagDistributionMeter = createReactClass({
 
   render() {
     return (
-      <div className="distribution-graph">
-        <h6>
-          <span>{this.props.tag}</span>
-        </h6>
+      <DistributionGraph>
+        <Tag>{this.props.tag}</Tag>
         {this.renderBody()}
-      </div>
+      </DistributionGraph>
     );
   },
 });
 
+const DistributionGraph = styled('div')`
+  position: relative;
+  font-size: 13px;
+  margin-bottom: 10px;
+`;
+
+const Tag = styled('div')`
+  position: relative;
+  font-size: 13px;
+  margin: 10px 0 8px;
+  font-weight: bold;
+  z-index: 5;
+  line-height: 1;
+`;
+
+const Description = styled('span', {shouldForwardProp: p => isPropValid(p)})`
```

Comment:
I think the `isPropValid` fat arrow function here is the default for shouldForwardProp. You would only need to do this if you were passing in some props that are usually passed to the DOM and then excluding them

eg

```js
{shouldForwardProp: p => p !== 'disabled' && isPropValid(p)}
```

Reply:
hmm. I guess I'm not sure how to implement the default then? It is for sure passing `first` and `last` to the underlying html element when I only do `styled('div')`. When I add `isPropValid` like the above it then filters out `first` and `last`.

Reply:
something like `shouldForwardProp: isPropValid`?